docs/topics/api/reviewers.rst

```diff
@@ -351,12 +351,22 @@ These endpoints allow you to draft comments that can be submitted through the re
     :<json string comment: The comment that is being drafted as part of a review.
     :<json string filename: The filename this comment is related to (optional).
     :<json int lineno: The line number this comment is related to (optional). Please make sure that in case of comments for git diffs, that the `lineno` used here belongs to the file in the version that belongs to `version_id` and not it's parent.
-    :<json int canned_response: The id of the draft comment (optional).
+    :<json int canned_response: The id of the :ref:`canned response <reviewers-canned-response-detail>` (optional).
+    :>json int id: The id of the draft comment object.
+    :>json string comment: The comment that is being drafted as part of a review. Specific to a line in a file.
+    :>json string|null filename: The filename a specific comment is related to.
```

Comment:
Could we just indicate the same object response as a `GET` is returned?  Duplicating it here and for the `PATCH` isn't very DRY and could end up with inconsistencies for any future changes in the serializer.

Reply:
Fixed in 
a3509d5